bigquery_etl/glam/generate.py

```diff
@@ -0,0 +1,161 @@
+    if not dataset_path.is_dir():
+        raise NotADirectoryError(f"path to {dataset_path} not found")
+
+    # curry functions for convenience
+    template = partial(
+        from_template, environment=env, dataset_path=dataset_path, args=args
+    )
+    view = partial(template, QueryType.VIEW)
+    table = partial(template, QueryType.TABLE)
+    init = partial(template, QueryType.INIT)
+
+    [
+        table(
+            "latest_versions_v1",
+            **dict(source_table="org_mozilla_fenix_stable.baseline_v1"),
+        ),
+        init(
+            "clients_scalar_aggregates_v1",
+            **models.clients_scalar_aggregates(
+                source_table=(
+                    f"glam_etl.{args.prefix}_view_clients_daily_scalar_aggregates_v1"
+                ),
+                destination_table=(
+                    f"glam_etl.{args.prefix}_clients_scalar_aggregates_v1"
+                ),
+            ),
+        ),
+        table(
+            "clients_scalar_aggregates_v1",
+            **models.clients_scalar_aggregates(
+                source_table=(
+                    f"glam_etl.{args.prefix}_view_clients_daily_scalar_aggregates_v1"
+                ),
+                destination_table=(
+                    f"glam_etl.{args.prefix}_clients_scalar_aggregates_v1"
+                ),
+            ),
+        ),
+        init(
+            "clients_histogram_aggregates_v1",
+            **models.clients_histogram_aggregates(parameterize=True),
+        ),
+        table(
+            "clients_histogram_aggregates_v1",
+            **models.clients_histogram_aggregates(parameterize=True),
+        ),
+        table(
+            "clients_scalar_bucket_counts_v1",
+            **models.clients_scalar_bucket_counts(
+                source_table="glam_etl.fenix_clients_scalar_aggregates_v1"
+            ),
+        ),
+        table(
+            "clients_histogram_bucket_counts_v1",
+            **models.clients_histogram_bucket_counts(),
+        ),
+        table(
+            "probe_counts_v1",
+            query_name_prefix="clients_scalar",
+            **models.probe_counts(
+                source_table="glam_etl.fenix_clients_scalar_bucket_counts_v1",
+                is_scalar=True,
+            ),
+        ),
+        table(
+            "probe_counts_v1",
+            query_name_prefix="clients_histogram",
+            **models.probe_counts(
+                source_table="glam_etl.fenix_clients_histogram_bucket_counts_v1",
+                is_scalar=False,
+            ),
+        ),
+        table(
+            "scalar_percentiles_v1",
+            **models.scalar_percentiles(
+                source_table="glam_etl.fenix_clients_scalar_aggregates_v1"
+            ),
+        ),
+        view("view_clients_daily_scalar_aggregates_v1"),
+        view("view_clients_daily_histogram_aggregates_v1"),
+        table("histogram_percentiles_v1"),
+        view("view_client_probe_counts_v1"),
+    ]
```

Comment:
out of curiousity, why are the square brackets needed?

Reply:
It's actually not necessary, it's just collecting the results in a list before throwing it away. I was thinking of using this for building up a mermaid diagram in the future.